common/app/common/configuration.scala

```diff
@@ -526,7 +526,6 @@ class GuardianConfiguration extends Logging {
       // this is a bit of a convoluted way to check whether we actually have credentials.
       // I guess in an ideal world there would be some sort of isConfigued() method...
       try {
-        val creds = provider.getCredentials
```

Comment:
This looks like a side-effect that we want to keep too - I suppose it could just be

    provider.getCredentials